src/js/controllers/backup.js

```diff
@@ -1,69 +1,125 @@
 'use strict';
 
-angular.module('copayApp.controllers').controller('wordsController',
-  function($rootScope, $scope, $timeout, profileService, go, gettext, confirmDialog, notification, bwsError, $log) {
+angular.module('copayApp.controllers').controller('backupController',
+  function($rootScope, $scope, $timeout, $log, $state, $compile, go, lodash, profileService, gettext, bwcService, bwsError) {
 
-    var msg = gettext('Are you sure you want to delete the backup words?');
-    var successMsg = gettext('Backup words deleted');
     var self = this;
-    self.show = false;
     var fc = profileService.focusedClient;
+    var customWords = [];
+    var mnemonic = null;
 
-    if (fc.isPrivKeyEncrypted()) self.credentialsEncrypted = true;
-    else {
-      setWords(fc.getMnemonic());
+    function init() {
+      $scope.passphrase = '';
+      resetAllButtons();
+      customWords = [];
+      mnemonic = null;
+      self.xPrivKey = null;
+      self.step1 = true;
+      self.step2 = false;
+      self.step3 = false;
+      self.step4 = false;
```

Comment:
Please, use a single numeric variable `self.step` and redefine the `goToStepN()` function to accept `N` as an argument.